README.md

```diff
@@ -387,6 +387,19 @@ stopped before the certificate can be renewed and then started again once
 renewal is finished. Otherwise, nothing will happen so it's safe to leave it
 running daily.
 
+### Maintenance mode
+
+While most operations performed within this Ansible playbook would not require downtime,
+some may do. For example, database migrations may break without newer application code.
```

Comment:
@jpmjpmjpm let's leave the example off for now because in general database migrations should not require maintenance mode and I don't want to confuse any potential users of the playbook. What do you think about removing the above two lines and starting with "The playbook contains a..." - this way we're simply giving users instructions on maintenance mode and letting them decide when to use it?